### PR TITLE
Fix/remove comments

### PR DESCRIPTION
**Pull request by mathild3r**
- [X] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [X] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

_Explain why this PR exists_
The customer would like the comment to be removed when there isn't any vuln left.
They also would like a debug mode

### Notes for the reviewer

_Instructions on how to run this locally, background context, what to review, questions…_

### More information

- [Link to documentation](https://github.com/snyk/snyk-prevent-gh-commit-status/wiki/)

### Screenshots

_Visuals that may help the reviewer_

### CODE REVIEW

File: src/lib/github/prComments.ts [hunk 1]
```diff
@@ -145,3 +150,82 @@ export const createPrComment = async (
...
+  commentUrlList.forEach(async url => {
+    debug(`Delete comment at url: ${url}`)
+    ghResponse = await ghClient.delete(
+      url);
+  });
+
+  return 
+}
+
+export const prCommentExist = async (
```
Comment (aarlaud) [inline src/lib/github/prComments.ts]: This function feels unnecessary. If there are pr comments that exist we'll delete them anyways, so you can just call delete which already pulls the comments to know which one(s) to delete. No need to do this work twice.

File: src/lib/index.ts [hunk 2]
```diff
@@ -94,7 +99,7 @@ const main = async () => {
           let shouldCommentPr = false;
           let noBaseline = false;
           
-          if (snykDeltaResults.result > 0 && ghPRNumber) {
+          if (snykDeltaResults.result === 1 && ghPRNumber) {
```
Comment (aarlaud) [inline src/lib/index.ts:102]: Why is this change required? What does it do or not do?

Comment (mathild3r) [reply src/lib/index.ts:102]: Like that we send a comment on the PR only if there are issues. if snyk-delta result with code 2 is if snyk-delta failed and we are sending an comment with the generic title : 'Vulnerabilities report for commit number...'

Comment (aarlaud) [reply src/lib/index.ts:102]: Ok

**Review by aarlaud — APPROVED** [review]
